build/pr-pipeline.yml

```diff
@@ -311,8 +317,5 @@ stages:
   - testStu3
   - testR4
   - testR5
-  - testAksR4Cosmos
-  - testAksR4Sql
-  - testAksR4SqlContainer
```

Comment:
If enableAks is true the cleanup still needs to wait on these jobs.

Reply:
Yep, don't have a good solution for this yet. They need to be there, but also they cause the entire step to be skipped at the moment...The AKS builds need some more work

Reply:
Will look at something like this: https://elanderson.net/2020/05/azure-devops-pipelines-depends-on-with-conditionals-in-yaml/